docs/USING_PRO.md

```diff
@@ -615,7 +611,7 @@ The parser takes tokens as input and calls the renderer functions.
 
 <h2 id="extend">Access to Lexer and Parser</h2>
 
-You also have direct access to the lexer and parser if you so desire.
+You also have direct access to the lexer and parser if you so desire. The `options` parameter is the same options that `marked.use` takes, documented [here](/using_advanced#options).
```

Comment:
It actually isn't the same as what `marked.use` accepts. `marked.use` accepts most options but some are different. For example, renderer doesn't need to extend marked.Renderer, it can just be an object (as explained in the bottom of https://marked.js.org/using_pro#use).

The lexer and parser options are the same as passed to  marked.setOptions() except they have to be full options objects, they don't get merged with the current or default options.

Reply:
Thanks for the updated explanation, I've tried to fix the docs accordingly.